runtime/lua/lsp/config/autocmds.lua

```diff
@@ -0,0 +1,14 @@
+local autocmds = require('lsp.autocmds')
```

Comment:
`lsp/config/autocmds.lua` is too much nesting. `lsp/config.lua` should be enough--if it gets too big that suggests we didn't factor out enough common utilities into `nvim` core library.